### DISCUSSION TITLE
Calculated String including a # prefix

### DISCUSSION THREAD

**Discussion by Gaztab** [microsoft-search/pnp-modern-search#2337] [Q&A] [answered]
Hi, 
I have made a calculated column in my SPO document library. The column looks at a date column to calculate if something is "Expired" or "Current" based off '<Today()'. This works fine.
I have mapped this column to a refinableString and then added this to my PnP search results to display in the card layout. 
Its appearing, but the results are 'string;#Expired' and 'string;#Current'. Why is this happening? How can I remove the unwanted 'string;#' part of this? 
Is this because its a calculated column and I need to treat this differently some how?

Any help on this would be appreciated. 

Thank you in advance.

**Comment by wobba** [comment, marked as answer]
That's how those columns are indexed. Try to use the  `remove` helper to manipulate the string. Something like:
```{{remove RefinableString00 "string;#"}}```

See https://github.com/helpers/handlebars-helpers#remove

**Reply by Gaztab** [reply]
This worked a treat, thank you very much.
